english/lucidream/luciddream.md

```diff
@@ -15,3 +15,5 @@ Yells and become super saiyan
 Flys to the moon
 
 [Turn into a potato](potato/potato.md)
+
+See what it feels like to [boil](boil/boil.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.